src/main/java/org/opensearch/security/support/ConfigConstants.java

```diff
@@ -120,6 +120,9 @@ public class ConfigConstants {
 
     public static final String OPENDISTRO_SECURITY_USER_INFO_THREAD_CONTEXT = OPENDISTRO_SECURITY_CONFIG_PREFIX + "user_info";
 
+    public static final String OPENDISTRO_SECURITY_IMPERSONATION_INITIATING_USER = OPENDISTRO_SECURITY_CONFIG_PREFIX
+        + "impersonation_initiating_user";
```

Comment:
wdyt about shortening this to `initiating_user`? This value will be set regardless of impersonation is being used or not and would be used for holding the identity of the original caller.

Reply:
Yes, I renamed it.

Also, one new integration test failed on windows build. It also failed on my local machine. It looks like there is a race condition in the test case, so I added a few retries.

Reply:
Just triggered the tests to run again